### PR TITLE
M3-449 Linode Watchdog Panel

### PR DESCRIPTION
**Pull request by rjstires**
## Purpose
Provide means to enable/disable Lassie/Watchdog.

## Note
Requires API release 53.3

### CODE REVIEW

File: src/features/linodes/LinodesDetail/LinodeSettings/LinodeWatchdogPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+  toggleWatchdog = (e: React.ChangeEvent<HTMLElement>, value: boolean) => {
+    this.setSubmitting(true);
+
+    updateLinode(this.props.linodeId, { watchdog_enabled: value })
+      .then((response) => {
+        this.setSubmitting(false);
+        this.setSuccess(`Watchdog succesfully ${value ? 'enabled' : 'disabled.'}`);
+      })
+      .catch(() => {
```
Comment (martinmckenna) [inline src/features/linodes/LinodesDetail/LinodeSettings/LinodeWatchdogPanel.tsx]: `this.state.submitting` will have to be reset upon failure

**Comment by martinmckenna** [comment]
@alioso This panel needs a breakpoint for smaller screens. The toggle is overlapping

screen shot 2018-06-22 at 8 50 24 am

**Comment by rjstires** [comment]
@alioso can you handle that breakpoint for me?

**Comment by asauber** [comment]
updated breakpoints, again @martinmckenna

**Pull request merged**